frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -45,30 +45,36 @@ TabPanel.propTypes = {
   value: PropTypes.any.isRequired,
 };
 
-// TODO: fetch alerts from backend and save in arrays.
-// Intended plan for reference: 
-// const allAlerts = [id1, id2, id3] where id'x' is the id of AlertX
-// const alertsMap = {id1: <Alert1>, 2: <Alert2>, 3: <Alert3>, ...};
-// const unresolvedAlerts = [0, 1] which stores the indices of the alerts in allAlerts
-// const resolvedAlerts = [2]
-const allAlerts = [0, 1, 2, 3, 4, 5, 6];
-const unresolvedAlerts = [0, 1, 2, 3];
-const resolvedAlerts = [4, 5, 6]
-
 // TODO: create constants file. ref: https://stackoverflow.com/questions/39036457/react-create-constants-file.
 const tabLabels = {
   UNRESOLVED: 0,
   RESOLVED: 1,
   ALL: 2,
 };
 
+/*
+ * Helper function to create Date from Timestamp model.
+ */
+const createDate = (timestampDate) => {
+  return new Date(
+    timestampDate.date.year, timestampDate.date.month - 1, timestampDate.date.day, 0, 0, 0, 0)
```

Comment:
also nit/optional and not super relevant for this specific project, but just noting that timezone differences sometimes can be very problematic/buggy in a production environment and is a likely cause of bugs. I assume here that the timestampDate sent from the java servlet backend doesn't have a related timezone with it, and so when you display it to the UI there is no choice.

In the beginning it might be good to just stick with a single timezone choice (depending on how the models/datastore stores this), and just use that one to reduce complexity of handling with differing timezones./

Reply:
okay, I'll sync with Melody on timezone